how-to/how-to-import-kata-logs-with-fluentd.md

```diff
@@ -185,7 +185,7 @@ in Kibana:
 ![Kata tags in EFK](./images/efk_syslog_entry_detail.png).
 
 We can however further sub-parse the Kata entries using the
-[Fluentd plugins](https://docs.fluentbit.io/manual/parser/logfmt) that will parse
+[Fluentd plugins](https://docs.fluentbit.io/manual/pipline/parser/logfmt) that will parse
```

Comment:
The correct URL should be `https://docs.fluentbit.io/manual/pipeline/parsers/logfmt`.

Reply:
OK @liubin